assets/queries/terraform/aws/elasticache_replication_group_not_encrypted_at_rest/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "76976de7-c7b1-4f64-a94f-90c1345914c2",
+  "queryName": "ElastiCache Replication Group Not Encrypted At Rest",
+  "severity": "MEDIUM",
+  "category": "Encryption",
+  "descriptionText": "Check if ElastiCache Replication Group encryption is disabled at Rest",
```

Comment:
I think this description could be something like: "ElastiCache Replication Group encryption should be enabled at Rest", what do you think?